client/src/resources/lang/en_us.json

```diff
@@ -463,131 +463,131 @@
+    "wiki.article.standard.innocent.text": "Innocent has multiple meanings.\n- A result the Detective can get usually indicating the player they visited is a townie.\n- Voting for someone to not be lynched during judgement phase. If there are more guilty votes than innocent ones, the player on trial is lynched.",
     "wiki.article.standard.suspicious.title": "Suspicious",
-    "wiki.article.standard.suspicious.text": "A result the Detective can get usually indicating the person they visited is evil.",
+    "wiki.article.standard.suspicious.text": "A result the Detective can get usually indicating the player they visited is evil.",
     "wiki.article.standard.guilty.title": "Guilty",
-    "wiki.article.standard.guilty.text": "Voting for someone to be lynched during a Judgement phase. If there are more Guilty votes, the player on trial is lynched.",
+    "wiki.article.standard.guilty.text": "Guilty counts as a vote for someone to be lynched during a Judgement phase. If there are more Guilty votes, the player on trial is lynched.",
     "wiki.article.standard.abstain.title": "Abstain",
-    "wiki.article.standard.abstain.text": "Abstaining is not voting during a Judgment phase. If all players Abstain, then the player on trial is not lynched.",
+    "wiki.article.standard.abstain.text": "Abstaining is when a player does not vote during a Judgment phase. If all players Abstain, then the player on trial is not lynched.",
     "wiki.article.standard.cleaned.title": "Cleaned",
     "wiki.article.standard.cleaned.text": "A player who is cleaned had their role and alibi erased from the graveyard. A Janitor can visit people on the night that they die to clean their grave.",
     "wiki.article.standard.evil.title": "Evil",
-    "wiki.article.standard.evil.text": "Has many meanings, Usually means a role is non-town. A psychic can recieve evil visions.",
+    "wiki.article.standard.evil.text": "If a role is evil, it means a role is non-town. A psychic can receive evil visions to learn who is not on town.",
     "wiki.article.standard.roleLimit.title": "Role Limit",
-    "wiki.article.standard.roleLimit.text": "A roles limit is the maximum number of that role that can be generated at the start of a regular game. If there is no limit, then there can be infinite. Because the veteran has a role limit of one, there usually can not be more than one veteran",
+    "wiki.article.standard.roleLimit.text": "A roles limit is the maximum number of that role that can be generated at the start of a regular game. If there is no limit, there can be any number of that role. Because the veteran has a role limit of one, there can not be more than one veteran unless manually forced into the role list or an amnesiac becomes one",
```

Comment:
I want to change this later, so its fine for now. Up to you

Reply:
We will change it later